<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgba(245, 221, 178, 1.0) ;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="13" column="2">
     <spacer name="horizontalSpacer_15">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="6">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="14">
     <spacer name="horizontalSpacer_16">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="7">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="6" colspan="9">
     <widget class="QLineEdit" name="leNombre">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
      </property>
      <property name="placeholderText">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6" colspan="9">
     <widget class="QLineEdit" name="leApaterno">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
      </property>
      <property name="placeholderText">
       <string>Apellido paterno</string>
      </property>
     </widget>
    </item>
    <item row="10" column="8" colspan="5">
     <widget class="QPushButton" name="btnRegistrar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(155, 88, 43, 1.0);
    color: white;
	padding: 8px;
    border-radius: 5px;
    transition: all 200ms;
}
QPushButton:hover {
	padding: 12px;
    background-color: rgba(209, 138, 91, 1.0) ;
}
</string>
      </property>
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="6" colspan="9">
     <widget class="QLineEdit" name="leRfc">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
      </property>
      <property name="placeholderText">
       <string>RFC</string>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <spacer name="horizontalSpacer_17">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="12">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="21">
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="9">
     <spacer name="horizontalSpacer_19">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="18">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="13">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="17">
     <spacer name="horizontalSpacer_18">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="6" colspan="9">
     <widget class="QLineEdit" name="leAmaterno">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
      </property>
      <property name="placeholderText">
       <string>Apellido materno</string>
      </property>
     </widget>
    </item>
    <item row="13" column="16">
     <spacer name="horizontalSpacer_14">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="3">
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="8">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="6" colspan="9">
     <widget class="QLineEdit" name="leNumero">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);

</string>
      </property>
      <property name="placeholderText">
       <string>Numero de trabajador</string>
      </property>
     </widget>
    </item>
    <item row="13" column="11">
     <spacer name="horizontalSpacer_20">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="8" colspan="5">
     <widget class="QLabel" name="mensaje">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>18</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(155, 88, 43, 1.0);;</string>
      </property>
      <property name="text">
       <string>mensaje</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="19">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="6" colspan="9">
     <widget class="QLineEdit" name="leEmail">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
      </property>
      <property name="placeholderText">
       <string>Correo electronico</string>
      </property>
     </widget>
    </item>
    <item row="13" column="10">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="5">
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="8" colspan="5">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(155, 88, 43, 1.0);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Registro Trabajadores</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="20">
     <widget class="QLabel" name="labelLink">
      <property name="maximumSize">
       <size>
        <width>59</width>
        <height>18</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(155, 88, 43, 1.0);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;a href=&quot;iniciar&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#95ef81;&quot;&gt;regresar&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
